beacon-chain/sync/subscriber.go

```diff
@@ -256,14 +306,16 @@ func (s *Service) subscribeDynamicWithSubnets(
 	topicFormat string,
 	validate pubsub.ValidatorEx,
 	handle subHandler,
+	digest [4]byte,
 ) {
-	base := p2p.GossipTopicMappings(topicFormat, 0)
-	if base == nil {
-		log.Fatalf("%s is not mapped to any message in GossipTopicMappings", topicFormat)
-	}
-	digest, err := s.forkDigest()
+	genRoot := s.cfg.Chain.GenesisValidatorRoot()
+	_, e, err := p2putils.RetrieveForkDataFromDigest(digest, genRoot[:])
 	if err != nil {
-		log.WithError(err).Fatal("Could not compute fork digest")
+		panic(err)
+	}
+	base := p2p.GossipTopicMappings(topicFormat, e)
+	if base == nil {
+		panic(fmt.Sprintf("%s is not mapped to any message in GossipTopicMappings", topicFormat))
```

Comment:
Again some panicking that perhaps could be replaced with error handling.

Reply:
same here again, if we can't find a valid base message it means this topic does not exist.